supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemAsyncJobExecutionActor.scala

```diff
@@ -358,7 +357,7 @@ trait SharedFileSystemAsyncJobExecutionActor
     def processSuccess(returnCode: Int) = {
       val successfulFuture = for {
         outputs <- Future.fromTry(processOutputs())
-      } yield SuccessfulExecutionHandle(outputs, returnCode, None)
+      } yield SuccessfulExecutionHandle(outputs, returnCode, Map.empty) //Ruchi: Need to fix this for SFS caching
```

Comment:
@kshakir I didnt get to show this to you when we discussed SFS-ABJEA but there's some debris from my PR that you'll see